packages/react-sdk/src/components/elements.ts

```diff
@@ -0,0 +1,34 @@
+export enum ELEMENTS {
```

Comment:
- we are not using ts enum
- this logic is gonna change a lot, because we want to make it all auto-generated, user only needs to add a component, the rest should be automated, meta will be optional